### DISCUSSION TITLE
Problem with image upload

### DISCUSSION THREAD

**Discussion by sekta2** [requarks/wiki#5914] [Error / Bug Report] [answered]
I don't know English that well so I'll use a translator.

When I just want to insert a picture, I see a window where you can upload a file, upload via a link (coming soon).
And just when I start to see all this, an unknown error appears, I think "okay" and start uploading a picture, after 100% an error occurs again, also unknown. I tried to create a folder and am getting the error:

**with recursive `ancestors` as (select `id`, `name`, `slug`, `parentId` from `assetFolders` where `id` = 0 union select `a`.`id`, `a`.`name`, `a`.`slug`, `a`.`parentId` from `assetFolders` as `a` inner join `ancestors` on `ancestors`.`parentId` = `a`.`id`) select * from `ancestors` - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'recursive `ancestors` as (select `id`, `name`, `slug`, `parentId` from `assetFol' at line 1**

**Comment by NGPixel** [comment, marked as answer]
You're using an outdated version of MySQL.

https://docs.requarks.io/install/requirements/mysql5

**Reply by sekta2** [reply]
I installed the new MySQL locally, i.e. in the same place where my wiki is installed. And everything is the same.

**Reply by NGPixel** [reply]
The error says otherwise. Which version of MySQL did you install?

**Reply by sekta2** [reply]
The old database died, I had to create it on the local host, but I can’t understand this

![изображение](https://user-images.githubusercontent.com/47900900/205732573-afb676f7-63a5-4735-a142-1857480286ef.png)

**Reply by sekta2** [reply]
> The error says otherwise. Which version of MySQL did you install?

I'm sorry, I can't even remember now because it's gone :(

**Reply by NGPixel** [reply]
Like I said, you're using an outdated MySQL version. 5.7 has limited support. Use MySQL 8.0 or later.
